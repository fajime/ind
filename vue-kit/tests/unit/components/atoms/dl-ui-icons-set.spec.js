import { mount } from '@vue/test-utils';
import DlUiIconsSet from '../../../../src/components/atoms/dl-ui-icons-set/index.vue';

describe('dl-ui-icon', () => {

  const wrapper = mount(DlUiIconsSet);

  it('Component Name', () => expect(DlUiIconsSet.name).toBe('dl-ui-icons-set'));

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
