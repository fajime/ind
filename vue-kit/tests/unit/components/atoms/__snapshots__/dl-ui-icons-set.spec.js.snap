// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dl-ui-icon renders correctly 1`] = `<!---->`;
